go-controller/pkg/ovn/namespace.go

```diff
@@ -199,7 +201,23 @@ func (oc *Controller) AddNamespace(ns *kapi.Namespace) {
 			nsInfo.hybridOverlayVTEP = parsedAnnotation
 		}
 	}
-
+	annotation = ns.Annotations[routingExternalGWsAnnotation]
+	if annotation != "" {
+		annotationOk := true
+		for _, v := range strings.Split(annotation, ",") {
+			parsedAnnotation := net.ParseIP(v)
+			if parsedAnnotation == nil {
+				klog.Errorf("Could not parse routing external gw annotation")
```

Comment:
include `v` in the error message. And probably the namespace name?

Reply:
done